sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/delete_key_operation.hpp

```diff
@@ -77,6 +77,13 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
         Azure::Core::Response<Azure::Security::KeyVault::Keys::DeletedKey> response);
 
   public:
+    /**
+     * @brief Get the #Azure::Core::Http::RawResponse of the operation request.
+     * @return A pointer to #Azure::Core::Http::RawResponse or null if the RawResponse does not
```

Comment:
When would it be null? When wouldn't  a response not exist?

Reply:
If you've extracted it already.  We don't allow Extracting it at the moment but we may want to support taking ownership of the contents at some point.

Reply:
Wouldn't we abort/throw at the point because that would be a dev-time bug in the calling code? It is odd that we'd be returning null raw responses in what I consider a buggy usage.

Reply:
We could throw if that's the design we want.  We want to dive deeper in the design.  As the code is written its correct.  BTW I filed a bug for our usage pattern on HttpTransports which has undefined behavior

Reply:
Ah I see, just saw the issue: https://github.com/Azure/azure-sdk-for-cpp/issues/1805